### ISSUE TITLE
Support for iOS 16.2

### ISSUE THREAD

**Issue by anitaspieker** [filsv/iOSDeviceSupport#158]

**Comment by marcosgalaviz-d** [comment]
yes please, any one knows when is going to be here IOS 16.2? Thanks

**Comment by bhadjajay23** [comment]
Please any one have device support file for iOS 16.2 please upload here. Need it urgent.

**Comment by bokhary3** [comment]
I tries suggestion where change folder 16.1 to 16.2 and it works

**Comment by chmiiller** [comment]
you can get 16.1 [here](https://github.com/iGhibli/iOS-DeviceSupport/blob/master/DeviceSupport/16.1%20(FromXCode_14.1(14B5024h)).zip) and just duplicate it and rename to 16.2

**Comment by anatolimacarovclimate** [comment]
What the min Xcode version I can use for 16.2 or what you have? I am trying with Xcode 13.2.1 and doesn't work.
Thanks

**Comment by anatolimacarovclimate** [comment]
I tried to toggle the [Developer Mode](https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device) on iPhone with iOS 16.2 but not successful. The iPad even doesn't have this menu in the Settings app. Possibly problem with not compatible MacBook operation system, my is Monterey 12.2.1v.

**Comment by fbaptista** [comment]
Hello. I created a pull request with latest images: https://github.com/filsv/iOSDeviceSupport/pull/160
Otherwise just pull them from the fork at: https://github.com/apptim/iPhoneOSDeviceSupport

**Comment by anatolimacarovclimate** [comment]
The developer mode and iOS 16.2 folder works from the Xcode 14.1v. The developer mode switcher appears when the device is connected to Xcode 14.1 and you trusted first connection.
@fbaptista thank you for the MR.
